### PR TITLE
UI pages

### PR DESCRIPTION
**Pull request by muskaanshah**
UI pages created for:
Home page
Explore page
Video cards which will be reused in history and liked video page 
Single video page
Modal which will open when button is clicked

As of now just the UI is created, functionality hasn't been added for any of these

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,20 @@
+import { Routes, Route } from "react-router-dom";
+
+import { Drawer } from "./components/Drawer/Drawer";
+import { Navbar } from "./components/Navbar/Navbar";
```
Comment (znjs) [inline src/App.js:4]: Can create an index for components also

**Review by znjs — APPROVED** [review]
- Clean code
- Good naming conventions used

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+			<span className="search">
+				<input
+					type="text"
+					className="input-text input-text-nav input-search mx-1"
+					placeholder="Search"
+				/>
+			</span>
+			<span class="avatar-default-sm borderradius-full bg-primary color-black fw-500 mx-1">
+				AT
+			</span>
```
Comment (znjs) [inline src/components/Navbar/Navbar.jsx:20]: can add pointer cursor to avatar icon

**Review by znjs — APPROVED** [review]

File: src/components/Navbar/Navbar.jsx (hunk 2 above)
Comment (manismk) [inline src/components/Navbar/Navbar.jsx:20]: change class to className

File: src/pages/Explore/components/Category.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+function Category({ item }) {
+	return (
+		<div className="input-wrapper color-white btn-category">
+			<label>
+				<input
+					type="checkbox"
+					value={item}
+					onChange={() => console.log("ticked")}
+				/>
```
Comment (manismk) [inline src/pages/Explore/components/Category.jsx:9]: can remove the console.log

File: src/pages/Explore/components/Time.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+function Time({ item }) {
+	return (
+		<div className="input-wrapper color-white btn-category">
+			<label>
+				<input
+					type="checkbox"
+					value={item}
+					onChange={() => console.log("ticked")}
+				/>
```
Comment (manismk) [inline src/pages/Explore/components/Time.jsx:9]: can remove the console.log

File: public/index.html [hunk 5]
```diff
@@ -25,7 +34,7 @@
     -->
 		<title>React App</title>
 	</head>
```
Comment (JowelTisso) [inline public/index.html:36]: Remember to change the app title

File: src/App.js [hunk 6]
```diff
@@ -1,8 +1,20 @@
...
     <div className="App">
       <header className="App-header">
-        <h1>New</h1>
+        <Navbar />
+        <Drawer />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/explore" element={<Explore />} />
+          <Route path="/singlevideo" element={<SingleVideo />} />
+        </Routes>
```
Comment (JowelTisso) [inline src/App.js:17]: You can extract routes in a separate file

**Review by manismk — APPROVED** [review]
Good naming convention and Good UI

**Review by JowelTisso — APPROVED** [review]
* Good work in making length variables 
* Please check the indentation of your file for readability
* Try to keep PR small

**Pull request merged**